.circleci/config.yml

```diff
@@ -1,7 +1,6 @@
 ---
 version: 2.1
 orbs:
-  slack: circleci/slack@3.4.2
```

Comment:
How is slack related to this PR? Just checking it's an intentional change :)

Reply:
it's not - a previous version of this CI step was attempting to use slack but it's not setup in hyperledger - so I removed that code - but forgot to remove the orb

Reply:
so yes it's an intentional change :)